### PR TITLE
Fix settings keys

### PR DESCRIPTION
**Pull request by ayoub-benali**

### CODE REVIEW

File: LSP-metals.sublime-settings [hunk 1]
```diff
@@ -33,11 +33,11 @@
   "settings": {
     "metals": {
       // When enabled, each method and variable that can have inferred types has them displayed.
-      "show-inferred-type": false,
+      "showInferredType": false,
```
Comment (tgodzik) [inline LSP-metals.sublime-settings:36]: Why the change? Both are actually correct.

Comment (ayoub-benali) [reply LSP-metals.sublime-settings:36]: Ah I didn't know ! 
My setup wasn't working and at some point I thought it was due to the key names :man_facepalming: 

I think they should be renamed anyway to match the metals doc which is camelcased
https://scalameta.org/metals/docs/integrations/new-editor#should-display-type-annotations-for-inferred-types

Comment (tgodzik) [reply LSP-metals.sublime-settings:36]: We can rename it for sure, I was mostly wondering if something was broken 😅

**Review by tgodzik — APPROVED** [review]

File: LSP-metals.sublime-settings (hunk 1 above)
Comment (ayoub-benali) [reply LSP-metals.sublime-settings:36]: Nothing is broken :D
